src/track/beatutils.h

```diff
@@ -68,6 +74,13 @@ class BeatUtils {
         const QVector<double> rawbeats, const int sampleRate,
         const int totalSamples, const double globalBpm);
 
+    static double samplesToFrames(const double samples) {
+        return floor(samples / mixxx::kEngineChannelCount);
```

Comment:
Why is the result rounded? The result should exactly match the argument, that's what the name of the function promises to do. It should not modify the resulting value by rounding.

Please introduce different functions for different purposes if needed.